src/tls13.c

```diff
@@ -9690,12 +9690,22 @@ static int SendTls13NewSessionTicket(WOLFSSL* ssl)
 #else
     extSz = EXTS_SZ;
 #endif
+    if (IsAtLeastTLSv1_3(ssl->version) &&
```

Comment:
By definition this function only gets called for TLS 1.3. No need to check for this.